checkov/bicep/parser.py

```diff
@@ -18,8 +18,8 @@ def parse(self, file_path: Path) -> tuple[BicepJson, list[tuple[int, str]]] | tu
 
         try:
             template = self.bicep_parser.parse(text=content)
-        except Exception:
-            logging.error(f"[bicep] Couldn't parse {file_path}", exc_info=True)
+        except Exception as e:
+            logging.warning(f"[bicep] Couldn't parse {file_path}: {e}")
```

Comment:
@rotemavni It's encouraged to use `exc_info=True` instead of printing the exception